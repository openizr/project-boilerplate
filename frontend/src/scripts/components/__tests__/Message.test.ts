/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/svelte';
import Message from 'scripts/components/Message.svelte';

describe('components/AppMessage', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(Message, { props: { label: 'test' } });
    expect(container.firstChild).toMatchSnapshot();
  });
});
