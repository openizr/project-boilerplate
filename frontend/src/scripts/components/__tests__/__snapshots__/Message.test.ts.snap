// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AppMessage renders correctly - basic 1`] = `
<div>
  <p>
    <img
      alt="test"
      class="svelte-1int8dr"
    />
     
    <img
      alt="test"
      class="svelte-1int8dr"
    />
     
    test
  </p>
</div>
`;
